### DISCUSSION TITLE
increase stunbaton's use per battery

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#4773] [Ideas and suggestions]
Due to the implementation of incremental stun (which is good), you can't stun someone with a fully charge battery since it only has two hits in it, suggestion to increase it so that 1 battery has more uses, but incremental stun should balance that out.

**Comment by Udrakan** [comment]
Yea, uses per battery could use some tweaking, but I like that you have to carry batteries with you and switch actively.

**Comment by CurePindal** [comment]
Fully agree. Since the change batteries are pretty useless in stun battons. Needs tweaking.

**Reply by Xeavas** [reply]
My original comment was when stun batons have 2 charges per battery after the implementation of incremental stun, the previous changes were a bandaid before incremental stun, and it was adjusted accordingly. Currently as of v12 I felt like it is alright and doesn't require change.

**Reply by CurePindal** [reply]
From what I see nobody uses stun batons any longer. 
They are useless dead weight in 1.0.8
